styles.css

```diff
@@ -421,10 +403,180 @@ body {
   padding-top: 10px;
 }
 
-.navigation ul {
+.navigation ul,
+#modalContainers {
   display: none;
 }
 
+.alert-Msg {
+  font-size: 12px;
+  font-weight: 600;
+  color: #fff;
+  padding-top: 12px;
+}
+
+/* Modal Design */
+#modalContainers {
+  background-color: rgba(100, 104, 140, 0.499);
+  font-family: 'Poppins', sans-serif;
+  list-style: none;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  backdrop-filter: blur(6px);
+  padding: 50px 14px 14px 14px;
+}
+
+.modalNavLi a {
+  font-size: 35px;
+  text-decoration: none;
+  color: #67798e;
+}
+
+#modalContainer a {
+  text-decoration: none;
+}
+
+.modalNav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  justify-content: space-between;
+  position: relative;
+  width: 100%;
+}
+
+.modalHeader {
+  padding: 8px 20px 20px 20px;
+  background-color: #fff;
+  width: 100%;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  position: relative;
+  height: 100%;
+  border-radius: 1rem;
+}
```

Comment:
Great job 🎉 addressing some of the changes requested by the previous reviewers. You're almost there 😉 but I noticed that while implementing the changes certain things got introduced. Your popup works and looks very well on mobile but the `border-radius` on the right side isn't quite there for the Desktop. See below 👇🏽 

![Screenshot 2023-02-04 at 11 21 18](https://user-images.githubusercontent.com/45739963/216778202-b0ee3d77-c8e4-49d0-9eae-f28f3ef075fa.png)